README.md

```diff
@@ -283,12 +283,17 @@ You'll see a view that looks like this:
 * When a unique job is enqueued, we'll atomically set a redis key that includes the job name and arguments and enqueue the job.
 * When the job is processed, we'll delete that key to permit another job to be enqueued.
 
-### Periodic Jobs
+### Periodic jobs
 
 * You can tell a worker pool to enqueue jobs periodically using a cron schedule.
 * Each worker pool will wake up every 2 minutes, and if jobs haven't been scheduled yet, it will schedule all the jobs that would be executed in the next five minutes.
 * Each periodic job that runs at a given time has a predictable byte pattern. Since jobs are scheduled on the scheduled job queue (a Redis z-set), if the same job is scheduled twice for a given time, it can only exist in the z-set once.
 
+## Paused jobs
+
+* You can pause all workers in a worker pool from processing any further jobs by setting a specific "paused" redis key (see `redisKeyJobsPaused`)
```

Comment:
This sounds like it is pausing a worker pool, when in fact it is pausing a queue.